app/docs/markdown/getting-started-with-dockstore.md

```diff
@@ -230,6 +230,64 @@ The `dockstore tool manual_publish` command can be used to manually register a t
 
     ------------------
 
+## Additional Information on Build Modes
+
+#### Fully-Automated
+
+How to create it:
+
+You create it by using the "Refresh All Tools" button.  This will scan through your Quay.io repositories and automatically register the tool on Dockstore.
+
+Requirements:
+
+- Using a Quay.io registry with Quay.io linked to Dockstore.
+
+Benefits:
+
+This build mode automatically adds versions to your tool with Quay tags and Git reference using the Quay.io build trigger.
+
+Limitations:
+
+- Unable to deregister tool
+- Unable to manually add versions
+- Cannot uniquely name the tool
+
+When to use:  
+
+Recommended when you are using a Quay.io registry, want a quick and easy way to register the tool, and want to avoid manually adding new versions to the tool.  Generally recommended for most tools.
+
+#### Manual
+
+How to create it:
+
+- Click Add Tool, use Quay.io registry, and change the tool name from default.
+
+Requirements:
+
+- Registry that has at least one tag
+- Token to the corresponding registry
+
+Limitations:
+
+- Have to manually the tool and also manually add each version (refresh will not work)
+
+When to use:
+
+Recommended when you're not using Quay.io or if someone else have the same tool name already and you want your own tool instead.  Also if you are not using build triggers.
+
+### Converting Between Build Modes
+
+Manual -> Fully-Automated:
+
+1. Deregister the manual tool
+2. Create a new Fully-Automated one
```

Comment:
I think this would be easily confused with "Add Tool", so this should be more explicit